src/components/SuggestedAction/SuggestedAction.tsx

```diff
@@ -88,12 +88,24 @@ const SuggestedAction = ({
         {dueDate &&
           <Typography variant="p3" sx={{ mt: 1 }}>Due on {dueDate}</Typography>
         }
-        <Box sx={{ mt: 2.5, display: 'flex', width: 'fit-content' }}>
-          <Button variant="text" onClick={ctaAction}>
+        <Box
+          sx={{
+            mt: 2.5,
+            display: 'flex',
+            width: 'fit-content',
+            flexWrap: { xs: 'wrap', sm: 'wrap' },
```

Comment:
do we need to break this down by breakpoint, if they're the same value in each case?